src/pages/about.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { Page } from '../types';
+import Layout from '../components/Layout';
+import Article from '../components/Article';
+import Footer from '../components/Footer';
+import '../styles/article-reader.scss';
+
+export default function AboutPage({ data }: Page): JSX.Element {
```

Comment:
Ive noticed that `pages/blog.tsx` and `pages/style-guide.tsx` both use arrow functions, but this page and the rest use conventional function syntax. What is the preferred method? If its the function syntax should we create a PR changing the 2 I just mentioned for consistency?